docs/firebase-bitrise-integration.md

```diff
@@ -0,0 +1,248 @@
+# EarlGrey 2 + Bitrise + Firebase Test Lab
+
+Firebase Test Lab runs EarlGrey 2 tests built on Bitrise.
```

Comment:
LG mostly. Can you please link to the Bitrise setup please?

Reply:
what do you mean link to the bitrise setup? I've included the complete YAML file in the doc. A user can copy and paste that into bitrise. Then they'll have the full setup.

The job itself is not public on bitrise due to security issues around code signing credentials being exposed on public jobs.

Reply:
I was thinking something like https://devcenter.bitrise.io/getting-started/getting-started-with-ios-apps/

In case a user might not even know what Bitrise is.

Reply:
Updated